lib/health_web/controllers/log_controller.ex

```diff
@@ -5,37 +5,40 @@ defmodule HealthWeb.LogController do
 
   action_fallback HealthWeb.FallbackController
 
-  plug Bodyguard.Plug.Authorize,
-       [
-         policy: Stats,
-         action: &__MODULE__.get_current_action/1,
-         user: &__MODULE__.get_current_user/1
-       ]
-       when action in [:index, :new, :create]
```

Comment:
I realized that using it as a plug kind of sucks if there is any sort of resource involved.